Command/CreateDatabaseDoctrineCommand.php

```diff
@@ -79,10 +79,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $error = false;
         try {
-            $shouldNotCreateDatabase = $ifNotExists && in_array($name, $tmpConnection->getSchemaManager()->listDatabases());
+            $cleanName = str_replace('`', '', $name);
```

Comment:
TBH I would rather consider not explicitly quoting the database identifier at all. IMO this is DBAL/user responsibility anyways. Some platforms even REQUIRE unquoted database identifiers like SQL Anywhere.